bibliopixel/project/types/defaults.py

```diff
@@ -0,0 +1,12 @@
+from . import channel_order, color, duration, gamma, ledtype
+
+FIELD_TYPES = {
+    'channel_order': channel_order,
```

Comment:
Well, yes, but there's nothing that can actually use it :P I'm fine with leaving it. Maybe in the future we'll make either c_order or channel_order work.